website/docs/docs/dbt-versions/core-upgrade/06-upgrading-to-v1.9.md

```diff
@@ -0,0 +1,109 @@
+- If anything goes wrong, or your schema changes, you can always "full-refresh", by running the same simple query that rebuilds the whole table from scratch.
+
+While this works for many use-cases, there’s a clear limitation with this approach: *Some datasets are just too big to fit into one query.*
+
+Starting in Core 1.9, you can use the new microbatch strategy to optimize your largest datasets  -- **process your event data in discrete periods with their own SQL queries, rather than all at once.** The benefits include:
+
+- Simplified query design: Write your model query for a single batch of data and no longer need manual filtering for determining "new" records. Use `event_time`, `lookback`, and `batch_size` configurations to generate necessary filters for you, making the process more streamlined and reducing the need for you to manage these details.
+- Independent batch processing: dbt automatically breaks down the data to load into smaller batches based on the specified `batch_size` and processes each batch independently, improving efficiency and reducing the risk of query timeouts. If some of your batches fail, you can use `dbt retry` to load only the failed batches.
+- Targeted reprocessing: To load a *specific* batch or batches, you can use the CLI arguments `--event-time-start` and `--event-time-end`.
+
+While microbatch is in "beta", this functionality is still gated behind an env var, which will change to a behavior flag when 1.9 is GA. To use microbatch:
+
+- Set `DBT_EXPERIMENTAL_MICROBATCH` to `true` in your project
+Currently microbatch is supported on these adapters with more to come:
+
+• postgres
+• snowflake
+• bigquery
+• spark
+### Snapshots improvements
+
+Beginning in dbt Core 1.9, we've streamlined snapshot configuration and added a handful of new configurations to make dbt **snapshots easier to configure, run, and customize.** These improvements include:
+
+- New snapshot specification: Snapshots can now be configured in a YAML file, which provides a cleaner and more consistent set up.
+- New `snapshot_meta_column_names` config: Allows you to customize the names of meta fields (for example, `dbt_valid_from`, `dbt_valid_to`, etc.) that dbt automatically adds to snapshots. This increases flexibility to tailor metadata to your needs.
+- `target_schema` is now optional for snapshots: When omitted, snapshots will use the schema defined for the current environment.
+- Standard `schema` and `database` configs supported: Snapshots will now be consistent with other dbt resources. You can specify where environment-aware snapshots should be stored.
+- Warning for incorrect `updated_at` data type: To ensure data integrity, you'll see a warning if the `updated_at` field specified in the snapshot configuration is not the proper data type or timestamp.
+
+### `state:modified` improvements
+
+We’ve made a number of improvements to `state:modified` behaviors to help reduce the risk of false positives/negatives, including:
```

Comment:
should we tie these to the behavior change flags mentioned below somehow?
`state_modified_compare_more_unrendered_values` and  `state_modified_compare_vars`

Reply:
Note that this was done in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/6184/commits/1365f9d2fe7c4b370875c308e321c0d5e9c82dd3)